response_operations_social_ui/__init__.py

```diff
@@ -76,4 +74,18 @@ def user_loader(user_id):
 
     setup_blueprints(app)
 
+    @app.before_request
+    def check_for_messages():  # pylint: disable=unused-variable
+        try:
+            redis_connection = redis.Redis(host=current_app.config['REDIS_HOST'], port=current_app.config['REDIS_PORT'])
+            maintenance_message = redis_connection.get(current_app.config['REDIS_MAINTENANCE_KEY'])
+            if maintenance_message:
+                maintenance_message = json.loads(maintenance_message)
+                logger.info('Maintenance message received from redis')
+                g.maintenance_message = maintenance_message
+        except redis.exceptions.ConnectionError as e:
```

Comment:
we should be more resilient to redis connection errors, in this case if we can't connect to redis then the app should carry on without showing the message instead of an error page appearing

Reply:
@davidmort addressed the resilience issue caused by flask-session depending on a redis connection. This will now fail quietly (with a log alert), which is also not ideal but at least the app won't fall over now 👍